tests/js/test-utils.js

```diff
@@ -115,6 +119,8 @@ const customRender = ( ui, options = {} ) => {
 		registry,
 		history,
 		waitForRegistry: () => act( waitForRegistry ),
+		waitForResolvingPromise: ( delay ) =>
```

Comment:
Similar to my other comment, the name is a bit ambiguous, and sounds more like our `AWAIT` action in the datastore which is essentially `async ( promise ) => promise`.

Ideally, we could enhance `waitForRegistry` to actually wait for everything as intended rather than introducing an arbitrary delay. Can we not simply advance the timers X amount before calling `await waitForRegistry()` to achieve the same?

Reply:
As mentioned this has been removed/replaced :+1: